tests/acceptance/features/apiSharingNg/sharedByMe.feature

```diff
@@ -1451,3 +1451,66 @@ Feature: resources shared by user
       }
     }
     """
+
+  @env-config
+  Scenario: user lists shared resources for deleted sharee
+    Given the config "GRAPH_SPACES_USERS_CACHE_TTL" has been set to "1"
```

Comment:
what this step is for? what willl be then step without this step

Reply:
the `env` is for how long to keep of the user cache. By default its 60 sec. But we cannot wait for 60 sec in  a test so keeping it least which is one to delete the users cache. When it is not used the in the list shared by a sharer, even after the sharee is deleted, its details are in the response which it should not be.